### PR TITLE
ember-a11y 2016-04-27

### PR DESCRIPTION
**Pull request by nathanhammond**
Notes from last week's meeting. Please review for correctness and completeness, respond with reactions if good, comments for places that need attention. 😄

/cc @ember-a11y/core

### CODE REVIEW

File: ember-a11y/2016-04/april-27.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  - Will do active research into markup after focus on fixing glaring Ember issues.
+  - Provide lots of examples.
+
+### Collection of a11y tools
+
+- We can totally steal some of approaches from other tools.
+  - https://github.com/prettydiff/a11y-tools
+  - https://twitter.com/burgessdryan/status/724384501326045184?s=09
+  - http://a11yproject.com/
+- We'll review each of these and regroup on things we should use or port to Ember.
```
Comment (ghost) [inline ember-a11y/2016-04/april-27.md:77]: @nathanhammond are we going to assign these as an action?

Comment (nathanhammond) [inline ember-a11y/2016-04/april-27.md:77]: My action was to assign those, but that'll be for a report three weeks from now and will come separately from these notes. :)

File: ember-a11y/2016-04/april-27.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+- We want to make sure that we're always out being visible in the community.
+- GEMConf - Nathan is speaking. Lightning talk.
+- EmberNYC - Get Suz to speak.
+- Ember ATX - Rob to find a presenter.
+- Shop Talk Show - Get Rob to speak?
+- EmberCamp London - Closing Keynote is Leonie Watson.
+
+### Relationships
+
+- Seattle, looking for somebdoy to work with Marcy Sutton.
```
Comment (noopkat) [inline ember-a11y/2016-04/april-27.md]: small typo in 'somebody'

File: ember-a11y/2016-04/april-27.md (hunk 1 above)
Comment (ghost) [inline ember-a11y/2016-04/april-27.md:77]: 👍

**Pull request merged**